x/smart/README.md

```diff
@@ -98,6 +101,32 @@ fallback:
   - socks5://[USERINFO]@[HOST]:[PORT]
 ```
 
+#### Psiphon config example
+
+> [!WARNING]
+> The Psiphon library is not included in the build by default because the Psiphon codebase uses GPL. To support Psiphon configuration please build using the [`psiphon` build tag](https://pkg.go.dev/github.com/Jigsaw-Code/outline-sdk/x/psiphon).
+> When integrating Psiphon into your application please work with the Psiphon team at sponsor@psiphon.ca
+
+JSON is a subset of YAML. If you have an existing psiphon JSON configuration file you can simply copy-and-paste it into your smart-proxy config.yaml file like so:
+
+```yaml
+fallback:
+  - psiphon: <YOUR_PSIPHON_CONFIG_HERE>
+```
+
+```yaml
+fallback:
+  - psiphon: {
+      "PropagationChannelId": "FFFFFFFFFFFFFFFF",
+      "SponsorId": "FFFFFFFFFFFFFFFF",
+      "DisableLocalSocksProxy" : true,
+      "DisableLocalHTTPProxy" : true,
+      # It's important to include an explicit timeout, otherwise an invalid connection will hang for a long time
```

Comment:
Let's remove this. The user should not edit the config and have to add anything.